client/src/components/profile/AuthenticationCard.vue

```diff
@@ -62,6 +62,7 @@ const props = defineProps<{
   state: AuthenticationCardState;
   authMethod: DeviceSaveStrategyTag | CustomDeviceSaveStrategyTag;
   disabled?: boolean;
+  isVerified?: boolean;
```

Comment:
I meant `isVerified` in my example xD In this case I don't really know what triggers that checkmark. Is it to show the current auth method? In this case, `isCurrent` or something like that.